pyproject.toml

```diff
@@ -18,7 +18,21 @@ exclude = [
 target-version = "py310"
 
 [tool.ruff.lint]
-extend-select = ["I"]
+select = [
+    # pycodestyle
+    "E",
+    "W",
+    # Pyflakes
+    "F",
+    # isort
+    "I",
+    # flake8-bugbear
+    "B",
+    # flake8-simplify
+    "SIM"
+    ]
+# Ignore line length errors
```

Comment:
`SIM108` doesn't seem to be related to line length. Can you break this up into multiple lines, move this existing comment to precede the `E501` line, and add a comment explaining why we're ignoring `SIM108`?

Reply:
Fixed in 4cf75dd14.